### PR TITLE
Release v2.2.1

### LINKED ISSUE #2: Unable to detect system theme
The application attempts to access `gsettings` directly to detect the system theme, which fails in the Flatpak environment since `gsettings` is not available in the `freedesktop` runtime. The recommended solution is to integrate [xdg-desktop-portal](https://flatpak.github.io/xdg-desktop-portal) using the [org.freedesktop.portal.Settings](https://flatpak.github.io/xdg-desktop-portal/docs/doc-org.freedesktop.portal.Settings.html) interface to read theme settings in a Flatpak-compatible way.

**Issue on NeoRegex:** https://github.com/NeoUtils/NeoRegex/issues/46

### PR DESCRIPTION
**Pull request by Irineu333**
- Fixes #2 (patch 1)
- Custom window: https://github.com/NeoUtils/NeoRegex/issues/32
- Small improvements
---
- Release v2.2.1: https://github.com/NeoUtils/NeoRegex/pull/51
- Release v2.2.0: https://github.com/NeoUtils/NeoRegex/pull/48

### CODE REVIEW

File: test.sh
Comment (Irineu333) [inline test.sh:1]: Is this allowed? I just wanted to make testing easier when I release a new version.

**Comment by Irineu333** [comment]
It’s not displaying the screenshots.

image

Maybe this is the reason:

image

File: test.sh
Comment (bbhtt) [reply test.sh:1]: yea it's fine

**Comment by bbhtt** [comment]
test builds don't have screenshots.

**Comment by bbhtt** [comment]
Seems fine, you can just merge no need to request review from me.

**Comment by Irineu333** [comment]
I fixed it for nothing haha. I thought it was strange since the lint hadn’t flagged anything.

**Pull request merged**